spinnaker/kork
# test(retrofit): check SpinnakerHttpException responseCode and message, and unexpectedError responseErrorBody

test(retrofit): check SpinnakerHttpException responseCode and message, and unexpectedError responseErrorBody

Tests:
* Check SpinnakerHttpException response code and message (added for more test coverage of SpinnakerHttpException)
* Verify that the unexpectedError does not have a responseErrorBody (added for more test coverage of RetrofitException)
